ocs_ci/ocs/resources/pod.py

```diff
@@ -203,7 +203,7 @@ def exec_ceph_cmd(self, ceph_cmd, format='json-pretty'):
 
         # For some commands, like "ceph fs ls", the returned output is a list
         if isinstance(out, list):
-            return [item for item in out if item]
+            return [item for item in out if not item == ""]
```

Comment:
I am up for dropping this line completely, let the test program handle what it needs rather than preprocess here?

Reply:
Fine with me, I thought there was an intention here to filter empty strings, but let remove it and run some testing to make sure it doesn't break anything.